pkg/internal/block_device.go

```diff
@@ -34,13 +41,14 @@ type BlockDevice struct {
 	PathByID   string        `json:"pathByID,omitempty"`
 	Serial     string        `json:"serial,omitempty"`
 	PartLabel  string        `json:"partLabel,omitempty"`
+	MajorMinor string        `json:"maj:min"`
 }
 
 // ListBlockDevices using the lsblk command
 func ListBlockDevices(exec Executor) ([]BlockDevice, error) {
 	// var output bytes.Buffer
 	var blockDeviceMap map[string][]BlockDevice
-	columns := "NAME,ROTA,TYPE,SIZE,MODEL,VENDOR,RO,RM,STATE,KNAME,SERIAL,PARTLABEL,FSTYPE"
+	columns := "NAME,ROTA,TYPE,SIZE,MODEL,VENDOR,RO,RM,STATE,KNAME,SERIAL,PARTLABEL,FSTYPE,MAJ:MIN"
```

Comment:
I see lsblk returns "loop" for the TYPE. Isn't that a better way that using Major numbers?

```sh-4.4# lsblk
NAME                                           MAJ:MIN RM   SIZE RO TYPE MOUNTPOINT
loop0                                            7:0    0     5G  0 loop 
sdb                                              8:16   0 893.8G  0 disk 
|-sdb1                                           8:17   0     1M  0 part 
|-sdb2                                           8:18   0   127M  0 part 
|-sdb3                                           8:19   0   384M  0 part /boot
`-sdb4                                           8:20   0 893.3G  0 part /sysroot
sr0                                             11:0    1  1024M  0 rom  
nvme0n1                                        259:0    0   1.5T  0 disk 
`-vg1-6c599a6b--632b--4d77--814c--61b7ecc4fd24 253:0    0     5G  0 lvm  
nvme1n1                                        259:1    0   1.5T  0 disk 
nvme2n1                                        259:2    0   1.5T  0 disk ```

Reply:
- yes, that could be an easier way
- I was fixated with major and minor number and failed to see this in plain sight

Reply:
done